### PR TITLE
refactor: metadata panel

### PR DESCRIPTION
**Pull request by noahamac**
Feelin much better about MetadataPanel now! 

It was the very first thing I did, so there was quite a bit of duplication. 

I'm relatively neglecting `QueryCharts/` in anticipation of an overhaul once `<Visualization />` lands.

### CODE REVIEW

File: src/components/DiagramFrame/MetadataPanel/common.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
+/*
```
Comment (noahamac) [inline src/components/DiagramFrame/MetadataPanel/common.tsx:1]: Here's the heart of the change. 

Applying styles for things that don't have defaultProps. Using an FC wrapper when no styles are needed

Comment (bryans99) [reply src/components/DiagramFrame/MetadataPanel/common.tsx:1]: Common as a name is a bit generic. Consider

utility_components
common_components
shared_components
metadata_components

**Review by bryans99 — CHANGES_REQUESTED** [review]
Looks good aside from common as a name for a file. Rename to reflect what it is

**Review by bryans99 — APPROVED** [review]
LGTM

**Pull request merged**